DependencyInjection/Configuration.php

```diff
@@ -49,6 +49,9 @@ public function getConfigTreeBuilder()
                         ->booleanNode('use_tls')
                             ->defaultFalse()
                         ->end()
+                        ->booleanNode('auto_connect')
+                            ->defaultFalse()
```

Comment:
Changes in the default behavior requires a new major version because this is a BC. No problem with that but document the new parameter in the README.md

Reply:
Sure, makes sense.  I have decided to change the default to `true` so it can be rolled into a patch release, rather than waiting for the next major BC release.

I have also updated the README pointing to the `7.0` version of the Adldap2 docs and fixed up a bit of the configuration in the README.